index.html

```diff
@@ -459,15 +459,17 @@ <h3>Claims</h3>
 
       <p>
 The data model for <a>claims</a> described above is powerful and can be used to
-express a large variety of statements. For example, whether or not someone is over
-the age of 21 may be expressed as follows:
+express a large variety of statements. For example, a claim specifying the date of birth of a subject
+can be used to prove that its age lies in an interval or is greater or less than a some value which is
+needed in use cases like proving drinking age or teenager discount or elderly discount;
+the holder has the flexibility to use the claim in any way that is applicable to that claim:
```

Comment:
I think the point of this statement (before the edit) was to indicate that an unbounded number of claims can be made using this model. It was not about how, for a given claim, additional statements can be proved. In fact, that is not a property of the model at all, but a property of a certain technology that can be used on top of the model (the model enables such technologies but does not provide the tech itself, it's just a model).

Therefore, we should change this section back to what it said before -- and then move these new edits just below the old text and talk about this additional powerful feature that is also compatible with (builds on top of) the model. So something like:

```
Additionally, selective disclosure schemes may also use claims expressed
in this model to prove additional statements about those claims. For example,
a claim specifying the date of birth of a subject can be used to prove that its
age lies...
```